test-suite/dune

```diff
@@ -1,12 +1,12 @@
 ; The easiest way to generate a portable absolute path is to use OCaml
 ; itself to print it
 (executable
- (name ocaml_pwd)
- (modules ocaml_pwd))
+ (name abspath)
```

Comment:
Renaming is to be avoided unless here is a good reason; in this case the old name is better so this needs to be reverted.